doc/_static/versions.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "name": "1.11 (devel)",
+        "name": "1.11 (dev)",
```

Comment:
-1 on changing the changelog folder name. It's not user-facing. This change is the only one I expected you to make.